docs/guides/using-map-matching.md

```diff
@@ -0,0 +1,39 @@
+In some cases, a developer may want to have the user stick to a very specific route, one that does not fit into the scope covered by the Mapbox Directions API. For example, a company would like to use their custom truck routing API but allow people to navigate on it with the Mapbox Navigation SDK. In this case, the Mapbox Map Matching API is an appropriate fit.
```

Comment:
Add this document to jazzy.yml so it appears under “Guides” in the table of contents.